src/components/CPU/BreakpointPanel.js

```diff
@@ -0,0 +1,209 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import './BreakpointPanel.css';
+import { ContextMenuTrigger } from 'react-contextmenu';
+import listeners from '../../utils/listeners';
+import BreakpointContextMenu from './BreakpointPanelContextMenu';

```

Comment:
nit: Would definitely prefer these both include or don't include Panel.  I'm not against shortening for imports, but not in the middle (i.e. `ContextMenu` would possibly also be okay, though I think I'd prefer the full name here.)

-[Unknown]

Reply:
Shortening the imports definitely wasn't my intent. This is just a result of bad renaming refactor.